extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/usergroup/UserGroupMemberUserMapper.java

```diff
@@ -19,10 +19,65 @@
 
 package org.apache.guacamole.auth.jdbc.usergroup;
 
+import java.util.Collection;
 import org.apache.guacamole.auth.jdbc.base.ObjectRelationMapper;
+import org.apache.ibatis.annotations.Param;
 
 /**
  * Mapper for the one-to-many relationship between a user group and its user
  * members.
  */
-public interface UserGroupMemberUserMapper extends ObjectRelationMapper<UserGroupModel> {}
+public interface UserGroupMemberUserMapper extends ObjectRelationMapper<UserGroupModel> {
+
+    /**
+     * Inserts rows as necessary to establish the one-to-many relationship
```

Comment:
This stuff about `RelatedObjectSet` is pretty abstract. It's not even clear just from reading this file what it's even about. Can you rephrase this (and any others) in terms of the specific relationships actually being modified?